doc/arkode/guide/source/Usage/MRIStep/MRIStepCoupling.rst

```diff
@@ -22,7 +22,36 @@ MRIStep supplies several built-in MIS, MRI-GARK, and IMEX-MRI-GARK methods, see
 coupling tables and their corresponding identifiers. Additionally, a user may
 supply a custom set of slow-to-fast time scale coupling coefficients by
 constructing a coupling table and attaching it with
-:c:func:`MRIStepSetCoupling`. The MRI coupling tables are stored in an
+:c:func:`MRIStepSetCoupling`. A given MRI coupling table can encode any of
+the MRI methods supported by MRIStep.  The family of MRI method encoded
+by the table is determined by an enumerated type, :c:enum:`MRISTEP_METHOD_TYPE`:
+
+.. c:enum:: MRISTEP_METHOD_TYPE
+
+   The MRI method family encoded by a :c:type:`MRIStepCoupling` table
+
+   .. c:enumerator:: MRISTEP_EXPLICIT
+
+      An explicit MRI-GARK method (does not support a slow implicit operator, :math:`f^I`).
+
+   .. c:enumerator:: MRISTEP_IMPLICIT
+
+      An implicit MRI-GARK method (does not support a slow explicit operator, :math:`f^E`).
+
+   .. c:enumerator:: MRISTEP_IMEX
+
+      An IMEX-MRK-GARK method.
+
+   .. c:enumerator:: MRISTEP_MERK
+
+      A explicit MERK method (does not support a slow implicit operator, :math:`f^I`).
+
+   .. c:enumerator:: MRISTEP_MRISR
```

Comment:
Having MRI twice in the name seems somewhat redundant. Perhaps just `MRISTEP_SR`?